nemo_text_processing/text_normalization/hu/verbalizers/electronic.py

```diff
@@ -64,21 +65,24 @@ def add_space_after_char():
 
         verbalize_characters = pynini.cdrewrite(graph_symbols | graph_digit, "", "", NEMO_SIGMA)
 
-        user_name = pynutil.delete("username: \"") + add_space_after_char() + pynutil.delete("\"")
+        user_name = pynutil.delete('username: "') + add_space_after_char() + pynutil.delete('"')
```

Comment:
add aliases for every string you insert to make it easier to edit later. (It also reduces the amount of changes that pop up on a review.)